dotnet/README.md

```diff
@@ -0,0 +1,149 @@
+# Documentation for dotnet distroless build
+
+## Image Contents
+
+This image definition contains a minimal Linux, dotnet-based [coreCLR](https://github.com/dotnet/core) runtime.
+
+Specifically, the image contains everything in the [base image](../base/README.md), plus:
+
+* coreCLR 2.0.0 Runtime and its dependencies.
+
+and
+
+```
+    base = "//base:base",
+    debs = [
+        packages["libcurl3"],
+        packages["libgcc1"],
+        packages["libstdc++6"],        
```

Comment:
I think you want `base="//cc:cc"` and you can drop the direct dependency on `libgcc1` and `libstdc++6`

Reply:
personally, i preferred the most baseline image to build off of (vs the cc) and manually declaring and adding stuff in but its not a big deal and changed to cc base

Reply:
FWIW, it is more than just cosmetic since it affects the layering of the final image.  Relying on `//cc` means you'll share those layers of the filesystem  with `/cc`-based images, which includes `//java` and `//python` based images.